Branch-SDK/Branch-SDK/BranchDeepLinkModel.h

```diff
@@ -0,0 +1,15 @@
+//
+//  BranchDeepLinkModel.h
+//  Branch-TestBed
+//
+//  Created by Parth Kalavadia on 5/15/17.
+//  Copyright © 2017 Parth Kalavadia. All rights reserved.
+//
+#import <Foundation/Foundation.h>
+#import "BranchDeepLinkingController.h"
+@interface BranchDeepLinkModel : NSObject
```

Comment:
Can you call this something more descriptive?  The word 'model' is really overused so it doesn't mean much.

Also, internal classes, classes that the SDK user is unlikely to ever use, start with `BNC` rather than `Branch`.

How about something like `BNCDeepLinkViewControllerInstance`.